Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/userSetup.py

```diff
@@ -33,56 +33,117 @@
 # -------------------------------------------------------------------------
 # built in's
 import os
-import sys
+#import sys
 import site
 import inspect
 import traceback
+from pathlib import Path
 import logging as _logging
+# -------------------------------------------------------------------------
 
-# -- DCCsi Extension Modules
-import azpy
-from azpy.constants import *
-from azpy.env_bool import env_bool
-from azpy.constants import ENVAR_DCCSI_GDEBUG
-from azpy.constants import ENVAR_DCCSI_DEV_MODE
 
-# To Do: needs to be updated to use dynaconf and config.py
-from azpy.env_base import _BASE_ENVVAR_DICT
+# -------------------------------------------------------------------------
+# global scope
+_MODULENAME = 'Tools.DCC.Maya.scripts.userSetup'
 
-# -- maya imports
-import maya.cmds as cmds
-import maya.mel as mel
-#from pymel.all import *
+__all__ = ['config',
+           'constants',
+           'setup',
+           'start',
+           'scripts']
+
+_LOGGER = _logging.getLogger(_MODULENAME)
 # -------------------------------------------------------------------------
 
 
 # -------------------------------------------------------------------------
+# Maya is frozen
+#_MODULE_PATH = Path(__file__)
+# https://tinyurl.com/y49t3zzn
+# module path when frozen
+_MODULE_PATH = Path(os.path.abspath(inspect.getfile(inspect.currentframe())))
+_LOGGER.debug('_MODULE_PATH: {}'.format(_MODULE_PATH))
+
+_DCCSI_TOOLS_MAYA_SCRIPTS_PATH = Path(_MODULE_PATH.parent.as_posix())
+_LOGGER.debug('_DCCSI_TOOLS_MAYA_SCRIPTS_PATH: {}'.format(_DCCSI_TOOLS_MAYA_SCRIPTS_PATH))
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS_MAYA = Path(_DCCSI_TOOLS_MAYA_SCRIPTS_PATH.parent)
+_PATH_DCCSI_TOOLS_MAYA = Path(os.getenv('PATH_DCCSI_TOOLS_MAYA', _PATH_DCCSI_TOOLS_MAYA.as_posix()))
+site.addsitedir(_PATH_DCCSI_TOOLS_MAYA.as_posix())
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS_DCC = Path(_PATH_DCCSI_TOOLS_MAYA.parent)
+_PATH_DCCSI_TOOLS_DCC = Path(os.getenv('PATH_DCCSI_TOOLS_DCC', _PATH_DCCSI_TOOLS_DCC.as_posix()))
+site.addsitedir(_PATH_DCCSI_TOOLS_DCC.as_posix())
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS = Path(_PATH_DCCSI_TOOLS_DCC.parent)
+_PATH_DCCSI_TOOLS = Path(os.getenv('PATH_DCCSI_TOOLS', _PATH_DCCSI_TOOLS.as_posix()))
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSIG = Path(_PATH_DCCSI_TOOLS.parent)
+_PATH_DCCSIG = Path(os.getenv('PATH_DCCSIG', _PATH_DCCSIG.as_posix()))
+site.addsitedir(_PATH_DCCSIG.as_posix())
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# now we have access to the DCCsi code and azpy
+from azpy.env_bool import env_bool
+from azpy.constants import *
+#from azpy.constants import ENVAR_DCCSI_GDEBUG
```

Comment:
Take out all unused/commented imports OR activate them and remove the wildcard import if you are using for reference purposes

Reply:
will fix in next PR, this whole file will get revamped